src/test/test-watchdog.c

```diff
@@ -27,12 +27,13 @@ int main(int argc, char *argv[]) {
                 t = 0;
 
         for (i = 0; i < count; i++) {
+                t = watchdog_runtime_wait();
+                log_info("Sleeping " USEC_FMT " microseconds...", t);
```

Comment:
Isn't this info already displayed by the previous call to `watchdog_setup()` ?

Reply:
watchdog_setup outputs the timeout value. The value from watchdog_runtime_wait is approximately half of that, and varies based on the exact time of the last ping.

The output now looks like this:

```
Using hardware watchdog 'SP5100 TCO timer', version 0, device /dev/watchdog0
Watchdog running with a timeout of 2s.
Sleeping 999999 microseconds...
Pinging...
Sleeping 1000000 microseconds...
Pinging...
Sleeping 1000000 microseconds...
Pinging...
```